src/components/sortControl/sortControl.test.tsx

```diff
@@ -0,0 +1,51 @@
+import { describe, expect, test } from 'vitest';
+import {
+  screen,
+  render,
+  fireEvent,
+  getByTestId,
+  waitFor,
+} from '@testing-library/react';
+import SortControl from './SortControl';
+import user from '@testing-library/user-event';
+import { vi } from 'vitest';
+
+let sortOptions: movieSortOption[];
+describe('Genre Select tests', () => {
+  beforeEach(() => {
+    sortOptions = ['Release Date', 'Title'];
+  });
+
+  test('check all sort options are added correctly', () => {
+    const SelectSortOptionHandler = vi.fn();
+    render(
+      <SortControl
+        sortOptions={sortOptions}
+        defaultSelection={sortOptions[0]}
+        onSelect={SelectSortOptionHandler}
+      />
+    );
```

Comment:
We dont need to duplicate this code in each test case. Extract it in the beforeEach hook!